install.rb

```diff
@@ -255,10 +255,13 @@ def install_binfile(from, op_file, target)
       tmp_file2 = Tempfile.new('hiera-wrapper')
       cwv = <<-EOS
 @echo off
-setlocal
-set RUBY_BIN=%~dp0
-set RUBY_BIN=%RUBY_BIN:\\=/%
-"%RUBY_BIN%ruby.exe" -x "%RUBY_BIN%hiera" %*
+SETLOCAL
+if exist "%~dp0environment.bat" (
+  call "%~dp0environment.bat" %0 %*
+) else (
+  SET PATH=%~dp0;%PATH%
+)
+ruby.exe -S -- hiera %*
```

Comment:
It should be the ruby that's on the path. Which, if this is called as a part of the puppet-agent package, it should be finding our ruby first, since we set our bindir to be the first thing on the path https://github.com/puppetlabs/puppet-agent/pull/512/files#diff-f19be7ccad3d10336f31489de24d09ddR20